addons/im_livechat/models/chatbot_script_step.py

```diff
@@ -42,6 +42,11 @@ class ChatbotScriptStep(models.Model):
         string='Only If', help='Show this step only if all of these answers have been selected.')
     # forward-operator specifics
     is_forward_operator_child = fields.Boolean(compute='_compute_is_forward_operator_child')
+    operator_expertise_ids = fields.Many2many(
+        "im_livechat.expertise",
+        string="Operator Expertise",
+        help="When forwarding live chat conversations, the chatbot will prioritize users with matching expertise.",
```

Comment:
Not really linked to chat bot anymore given the implementation of `_get_operator`.

Reply:
This is a field on chatbot script step, so seems fair to refer to chatbot in this context, no?